pkg/apis/migration/v1alpha1/model.go

```diff
@@ -2,6 +2,7 @@ package v1alpha1
 
 import (
 	"context"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Let's alias this as `metav1` instead of just `v1` so that it's clear what the alias refers to.

Reply:
Done